packages/core/src/readers/NotionReader.ts

```diff
@@ -59,13 +64,32 @@ export class NotionReader implements BaseReader {
     return this.crawl(rootPageId);
   }
 
+  /**
+   * Loads recursively the Notion database with the specified database ID.
+   * @param {string} databaseId - The Notion database ID from which to start the crawl
+   * @returns {Promise<Pages>} A Promise that resolves to a Pages object(Convertible with the `toDocuments` method)
+   */
+  async loadDatabasePages(databaseId: string): Promise<Pages> {
+    return this.dbCrawl(databaseId);
```

Comment:
does this also return `Pages`?

Reply:
Hm, I was giong to say yes as that's what the typescript types say in the editor. However, after digging in to the code to confirm I found that it will treat the page as a database if normal extraction fails:

https://github.com/TomPenguin/notion-md-crawler/blob/main/src/crawler.ts#L279-L280

So, it would seem this PR is unnecessary. It would be more efficient to avoid the work in the `try` branch but avoiding additional complexity is probably more important.